docker-pg-databases/simulateur.sql

```diff
@@ -9,19 +9,21 @@ CREATE TABLE Users (
 
 CREATE TABLE Sensor (
     id SERIAL PRIMARY KEY,
-    coords INT[2]
+    latitude FLOAT,
```

Comment:
double pas de float en java c'est nazoss, valable de partout en soit

```suggestion
    latitude double,
```